.github/workflows/check.yaml

```diff
@@ -8,26 +8,27 @@ jobs:
   check:
     runs-on: ubuntu-latest
     steps:
-
       - name: Checkout
         uses: actions/checkout@v2
 
       - uses: subosito/flutter-action@v2
         with:
+          flutter-version: "3.0.x"
+          channel: "stable"
```

Comment:
This is the default for this action - see [here](https://github.com/subosito/flutter-action/blob/0c3f14223a08fa950c8a4c00bcfb834e65744135/action.yml#L15).

Reply:
I've added those in a separate commit, because I thought it would be better to have both version and channel defined. I removed the commit, so now we only specify version